File: HW01/CPP/src/SoccerballNet.cpp
#include "../include/SoccerballNet.h"
#include <iostream>
#include <string>
using namespace std;

SoccerballNet::SoccerballNet(int size, double price) : OutdoorItem(price){
    this->size = size;
    this->price = price;
}

void SoccerballNet::display() {
    cout << "$" << this->price << " - This soccerball net is " << this->size << " feet wide";
}

File: HW01/PYTHON/project/SoccerballNet.py
from .OutdoorItem import OutdoorItem

class SoccerballNet(OutdoorItem):
    def __init__(self, price, size):
        super().__init__(price)
        self.size = size
             
    def display(self):
        print(f"${self.price} - This soccerball net is {self.size} feet wide", end="")